### DISCUSSION TITLE
Mobile site Pull Requests heading omits Review Requests

### DISCUSSION THREAD

**Discussion by romoore** [community/community#76420] [Pull Requests]
### Select Topic Area

Bug

### Body

The mobile PRs view has 3 headings (Created, Assigned, Mentioned), but is missing the fourth: Review Requests. I have to switch to the Desktop version, get to the page, then switch back each time.

![Screenshot_20231121_084802_Chrome](https://github.com/community/community/assets/1753199/e26ac4c4-5d93-4079-bcfb-d1d7221219a3)
![Screenshot_20231121_084809_Chrome](https://github.com/community/community/assets/1753199/f4a941ef-3f9e-4719-beb2-17eaec4844bf)
![Screenshot_20231121_084817_Chrome](https://github.com/community/community/assets/1753199/039008b0-8da6-4d98-acab-93fc09e3a978)

**Comment by andyzakUSI** [comment]
If you rotate your phone it will also show the Review Requests button without going into Desktop mode - but it's very annoying to have to do that.

**Reply by romoore** [reply]
It would, but that wouldn't be accessible to someone using the device in a fixed orientation. For example, mounted on a wheelchair. Hopefully they can fix this soon. :)
